src/Common/Core/Impl/Versions/VsVersion.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.Common.Core {
+    public static class VsVersion {
```

Comment:
We don't have general assembly for VS-specific things. Maybe we need to add one. Or use `#ifdef` in this particular case.